rules/docker-teamd.mk

```diff
@@ -1,13 +1,26 @@
 # docker image for teamd agent
 
-DOCKER_TEAMD = docker-teamd.gz
-$(DOCKER_TEAMD)_PATH = $(DOCKERS_PATH)/docker-teamd
+DOCKER_TEAMD_STEM = docker-teamd
+DOCKER_TEAMD = $(DOCKER_TEAMD_STEM).gz
+DOCKER_TEAMD_DBG = $(DOCKER_TEAMD_STEM)-$(DBG_IMAGE_MARK).gz
+
+$(DOCKER_TEAMD)_PATH = $(DOCKERS_PATH)/$(DOCKER_TEAMD_STEM)
+
 $(DOCKER_TEAMD)_DEPENDS += $(SWSS) $(LIBTEAMDCT) $(LIBTEAM_UTILS) $(REDIS_TOOLS)
+$(DOCKER_TEAMD)_DBG_DEPENDS = $($(DOCKER_CONFIG_ENGINE_STRETCH)_DBG_DEPENDS)
+$(DOCKER_TEAMD)_DBG_DEPENDS += $(LIBSAIREDIS_DBG) $(SWSS_DBG) $(LIBSWSSCOMMON_DBG)
```

Comment:
LIBSAIREDIS_DBG is not needed.

Reply:
can we get libteamd dbg symbols in the dbg docker?

Reply:
I don't see we getting/building libteam dbg symbols in (refer: rules/libteam.mk & src/libteam/Makefile)

I will drop LIBSAIREDIS_DBG

Reply:
If we need it and if it is available for 'fetching/building', we need to do that separately and add to DBG_DEPENDS